CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ elseif(UNIX)
     set(CMAKE_INSTALL_RPATH "./libs")
     set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lc++abi -lrt")
+    link_directories("/usr/local/lib")
```

Comment:
Not sure if we should commit absolute paths. Please take a look on PR #43. Even now if we have this open source, we shouldn't force paths for our specific envs. On other hand, this is a very common path on linux, shouldn't hurt if it stay

Reply:
I plan to replace with your other PR. I was not sure to commit this since it compiled before on Jenkins, but not on my machine. I can keep it locally until replaced.